lib/output.js

```diff
@@ -23,74 +23,102 @@ function overview(url, strategy, scores) {
   });
 
   ret.push({
-    label: 'Speed',
-    value: scores.SPEED.score
+    label: 'Performance',
+    value: reversePercentage(scores.categories.performance.score)
   });
 
-  if (scores.USABILITY !== undefined) {
-    ret.push({
-      label: 'Usability',
-      value: scores.USABILITY.score
-    });
-  }
-
   return ret;
 }
 
-function ruleSetResults(rulesets) {
+const fieldData = stats => {
   const ret = [];
 
-  for (const title in rulesets) {
-    if (Object.prototype.hasOwnProperty.call(rulesets, title)) {
+  for (const title in stats) {
+    if (Object.prototype.hasOwnProperty.call(stats, title)) {
       ret.push({
         label: title,
-        value: Math.ceil(rulesets[title].ruleImpact * 100) / 100
+        value: prettyMs(stats[title].percentile)
       });
     }
   }
 
   return sortOn(ret, 'label');
-}
+};
 
-function statistics(stats) {
+const getRules = (rules, group) => {
+  return rules.filter(rule => {
+    return rule.group === group;
+  });
+};
+
+const labData = lighthouseResult => {
   const ret = [];
 
-  for (const title in stats) {
-    if (Object.prototype.hasOwnProperty.call(stats, title)) {
-      ret.push({
-        label: title,
-        value: title.indexOf('Bytes') === -1 ? stats[title] : prettyBytes(Number(stats[title]))
-      });
-    }
-  }
+  const rules = getRules(lighthouseResult.categories.performance.auditRefs, 'metrics');
+
+  rules.forEach(rule => {
+    const {audits} = lighthouseResult;
+    ret.push({
+      label: audits[rule.id].title,
+      value: audits[rule.id].displayValue
+    });
+  });
 
   return sortOn(ret, 'label');
-}
+};
+
+const opportunities = lighthouseResult => {
+  const ret = [];
+
+  const rules = getRules(lighthouseResult.categories.performance.auditRefs, 'load-opportunities');
+
+  const oppRule = rules.filter(opportunityRule => {
+    const ruleDetails = lighthouseResult.audits[opportunityRule.id].details;
+    return ruleDetails.overallSavingsMs !== undefined && ruleDetails.overallSavingsMs > 0 && ruleDetails.type === 'opportunity';
+  });
+
+  oppRule.forEach(rule => {
```

Comment:
Use a `for-of` loop.

Reply:
Better in that way. I didn't want to modify too much in this first PR.